covsirphy/cleaning/jhu_complement.py

```diff
@@ -199,17 +203,54 @@ def _recovered_full(self, df):
         if df[self.R].max() > self.max_ignored:
             # Necessary if sum of recovered is more than 99%
             # of sum of recovered and infected when outbreaking
-            sel_1 = df[self.C] > self.max_ignored
+            sel_C1 = df[self.C] > self.max_ignored
+            sel_R1 = df[self.R] > self.max_ignored
             sel_2 = df[self.C].diff().diff().rolling(14).mean() > 0
-            s_df = df.loc[
-                sel_1 & sel_2 & (df[self.R] > 0.99 * (df[self.C] - df[self.F]))]
-            if s_df.empty:
+            sel_3 = df[self.R] < 0.01 * (df[self.C] - df[self.F]).rolling(14).mean()
```

Comment:
Please update the following comments or update the docstring of `_recovered_full()`.

```
# Necessary if sum of recovered is more than 99%
# of sum of recovered and infected when outbreaking
```

Reply:
Updated comment with
```
            # Necessary if sum of recovered is more than 99%
            # or less than 1% of sum of recovered minus infected when outbreaking
```